### DISCUSSION TITLE
Vertical gutter spacing inconsistent for children of grid row

### DISCUSSION THREAD

**Discussion by Ammamon** [quasarframework/quasar#15722] [General - Components / Directives / etc] [answered]
For grid row children, gutter working for horizontal spacing. But vertical spacing applied only the last few children in my case. Not understanding why the vertical gutter is not applied to the first few children. Please find below my code and display below.

![image](https://user-images.githubusercontent.com/7417777/232312128-a35bffbb-01c7-4335-b5ec-1ee20cb90a6d.png)

![image](https://user-images.githubusercontent.com/7417777/232310405-c641913c-1909-4dd7-a913-fe396da6e8bf.png)

**Comment by pdanpdan** [comment]
Make a codepen

**Reply by Ammamon** [reply]
I have created the code pen example.

https://codepen.io/ammamon/pen/jOeqJpQ

It worked there without the validation rules etc.

But in my actual code, when I add the below validation rules for certain fields, the vertical gutter setting is not applied.
:rules="[validateInputFieldRequirement]" lazy-rules 

The fields without the validation rule is following the vertical gutter setting.

The horizontal gutter setting is working correctly for all fields even with the validation rules.

See the snapshot of my actual code below.

Please help to solve this issue.

![image](https://user-images.githubusercontent.com/7417777/232322405-846b21fe-fe1d-480e-ac14-b147279826fe.png)

**Comment by Ammamon** [comment, marked as answer]
Solved the issue. Used hide-bottom-space attribute for the fields using the rules validation.
